docs/howto/variable.rst

```diff
@@ -0,0 +1,72 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Managing Connections
+====================
+
+Variables are a generic way to store and retrieve arbitrary content or
+settings as a simple key value store within Airflow. Variables can be
+listed, created, updated and deleted from the UI (``Admin -> Variables``),
+code or CLI.
+
+.. image:: ../img/variable_hidden.png
+
+See the :ref:`Variables Concepts <concepts-variables>` documentation for
+more information.
+
+Storing Variables in Environment Variables
```

Comment:
If I may add... there the previous paragraph explain what is Variable.
This paragraph explain how to store variable as Environment Variables (This is a **How to** but not **What is**)
It's not clear to me why to do that. What this functionality gives and when someone would need to use it.

Reply:
Can you add it in a separate PR? I just want to create a page in this PR based on the existing information. I would have added sections, but we should add similar information to other pages too, and changes to other pages are beyond the scope of this PR. WDYT?

Reply:
I was pointing that because I don't know the answers and there is no where in the docs where this info available.

Reply:
Environment variables are helpful because they can be set very easily in a containerized environment and their contents can be determined during deployment.
https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
It could also contain secrets.
https://kubernetes.io/docs/concepts/configuration/secret/
In the DAG file, you can also read environment variables, but the use of Airflow variables allows for their easier configuration in the development environment, e.g. access to UI, easier development - access to variable in Jinja, and thanks to setting Airflow variables using environment variables also easy deployment.